groups/bsl/bslstl/bslstl_inplace.h

```diff
@@ -37,6 +44,13 @@ namespace bsl {
 using in_place_t = std::in_place_t;
 using std::in_place;
 
+template <class TYPE>
+using in_place_type_t = std::in_place_type_t<TYPE>;
```

Comment:
I don't mind changing it. Can you elaborate on subtleties involving alias-declaration ?

Reply:
Thanks !
They're also simpler to write and I agree with the blanket preference suggestion. Btw, I introduced the `in_place_t` one too for no apparent reason, so I suggest I fix it since I'm already in this area.